action.yml

```diff
@@ -4,17 +4,21 @@
   GITHUB_TOKEN:
     description: "GitHub token to interact with the repository."
     required: true
-  OPENAI_API_KEY:
-    description: "OpenAI API key for GPT."
+  INFERENCE_API_KEY:
+    description: "OpenAI or Anthropic API key for LLM inference."
     required: true
-  OPENAI_API_MODEL:
-    description: "OpenAI API model."
+  MODEL:
+    description: "AI model to use for the code review."
     required: false
-    default: "gpt-4"
+    default: "gpt-4o-mini"
```

Comment:
The default model `gpt-4o-mini` seems to be a custom or non-standard model name. Ensure this is the correct identifier for the intended model, as it's not a commonly known OpenAI or Anthropic model name.